src/components/common/YourCompany/CompanyName/CorrectNameToNumber.vue

```diff
@@ -30,26 +29,33 @@ export default class CorrectNameToNumber extends Vue {
 
   @Action setNameRequest!: ActionBindingIF
 
-  @Getter getApprovedName!: string
-  @Getter getEntityType!: CorpTypeCd
+  @Getter getNameRequest!: NameRequestIF
   @Getter getBusinessId!: string
 
   // Local properties
   private correctToNumbered = false
 
+  get businessId (): string {
+    return this.getBusinessId && this.getBusinessId.substring(2)
```

Comment:
To change BC1234567 to 1234567. Ie,

![image](https://user-images.githubusercontent.com/10002889/179639673-97938732-3184-4dd4-83b1-53e9dfa19fab.png)